backend/src/routes/Registry/registry-controllers.ts

```diff
@@ -20,6 +21,17 @@ export const createRegistry: RequestHandler = async (req, res, next) => {
   }
 };
 
+export const getMyRegistry: AuthHandler = async (req, res, next) => {
+  try {
+    const userId = getUserId(req.user?.sub);
```

Comment:
I see you imported a util function, but you won't need it. 

```javascript
const userId = req.user?.sub;

// if there's no user, throw an error
if(!userId) throw createError(400, "No userId found"); // we can save a DB read by just throwing here
const registry = await Registry.findOne({ userId });
```

Reply:
That's all you need

Reply:
It doesn't really make sense to check the userId since it's already gone through the `verifyToken` middleware which should catch whether a user exists or not right?